libsplinter/src/network/peer_manager/peer_map.rs

```diff
@@ -20,7 +20,7 @@ use crate::collections::BiHashMap;
 use super::error::PeerUpdateError;
 
 // Default intial value for how long to wait before retrying a peers endpoints
-const INITIAL_RETRY_FREQUENCY: u64 = 10;
+pub const INITIAL_RETRY_FREQUENCY: u64 = 10;
```

Comment:
Making this pub has a bit of a code smell to it.  I think that either: the `PeerMap` should be initialized with this value (and it not being a constant) or the place the place where this is being set in the peer manager, should use a `replace` method, that will re-initialize those internal values.

Reply:
Ah yep i forgot to change that. I am going to have it passed into PeerMap.

Reply:
Added a commit for this